src/dsf/__init__.py

```diff
@@ -1,4 +1,8 @@
-SOCKET_DIRECTORY = "/run/dsf"
-SOCKET_FILE = "dcs.sock"
-FULL_SOCKET_PATH = SOCKET_DIRECTORY + "/" + SOCKET_FILE
+# path to unix socket file
+SOCKET_FILE = "/tmp/dsf/dsf.sock"
```

Comment:
I don't think you should replace the default location of the socket as it is on systems actually running DSF

Reply:
ups. fixed.